docs: OT-2 instruction manual revisions

jwwojak opened the pull request:
# Overview

Revise and migrate OT-2 instruction manual into the new documentation system. Flex instruction manual is the model.

- Revise: bring prose and contents up to current writing standards. 
- Remove: unnecessary contents, sections, and even chapters. 
- Migrate: get this stuff into MkDocs

Other changes: There should be no really new information added, unless something important is missing. This is old wine, a new bottle, and remove the sediment.

We will create separate branches from this for each chapter and merge those back into this branch after review.

** Edit sandbox link**: https://sandbox.docs.opentrons.com/docs-ot2-manual-revisions/ot-2/

References:

- [OT-2 Instruction Manual Revisions](https://docs.google.com/document/d/1SwAmYjZU2GpNNKTMhwRzN5tDGhpdsdufKAa1jPs-SKg/edit?usp=sharing)
- Original [OT-2 instruction manual](https://insights.opentrons.com/hubfs/Products/OT-2/OT-2R%20User%20Manual%20V1.0.pdf) for comparison. Also attached to JIRA, linked below.
- RTC-855

## Test Plan and Hands on Testing

I can haz wordz. Plz read.
![gudRigting](https://github.com/user-attachments/assets/537d0677-bd70-4674-909d-92687bce5a96)

## Changelog

2 new files for review:
- `introduction.md`
- `regulatory.md`

## Review requests

If some text is familiar or seems familiar, that's because it may be. Some OT-2 sections are similar to the Flex manual. In many cases, this project will replace that older content with text from the Flex manual, and just change "Flex" to "OT-2."

## Risk assessment

Low because this is documentation; medium because it is content an installed user-base relies on. Need to keep the revised manual useful for those folks.

jwwojak committed:
Created project space-just to test

Delete if the project shouldn't go forward.

jwwojak committed:
Sample sections with placeholders

Created sections to see what this could look like.

- placeholder section headers
- placeholder text

jwwojak committed:
Outline: intro

jwwojak committed:
Outline: safety and regulatory section

jwwojak committed:
Merge branch 'edge' into docs-ot2-manual-revisions

jwwojak committed:
Installation

jwwojak committed:
Outline for a few more sections

jwwojak committed:
Regulatory and compliance content

jwwojak committed:
Update regulatory.md

jwwojak committed:
Intro and regulatory sections morning commit

jwwojak committed:
introduction and regulatory sections

After this, we'll stop and make a branch from the draft pr.

Each chapter will be a branch.

jwwojak committed:
End of day commit: regulatory text

After this, these will be separate PRs.

jwwojak committed:
One more regulatory

jwwojak committed:
Intro and regulatory push

Moving on to other chapters

emilyburghardt commented on docs/ot-2/docs/appendicies.md:
```diff
@@ -0,0 +1,10 @@
+---
+title: "Opentrons OT-2: Appendicies"
```
appendices?

jwwojak replied:
That shouldn't be in this branch. Removing.

emilyburghardt commented on docs/ot-2/docs/index.md:
```diff
@@ -0,0 +1,14 @@
+---
+title: "Opentrons OT-2 Liquid Handler Instruction Manual"
```
should we just keep it "Opentrons OT-2 Instruction Manual"? 

the Flex manual title doesn't include "liquid handler"

jwwojak replied:
Yes, let's drop the "liquid handler" part. That's from the original OT-2 manual, but this change will align it more closely w/ flex.

emilyburghardt commented on docs/ot-2/docs/installation.md:
```diff
@@ -0,0 +1,25 @@
+
+### Requirements
+
+Warning: The Opentrons Liquid Handler can emit vibration while in operation. For
```
replace "Opentrons Liquid Hander" with OT-2 throughout section?

jwwojak replied:
This file shouldn't be in here. Just look at `introduction.md` and `regulatory.md`

emilyburghardt reviewed (commented):
Looking pretty good so far, Joe! Left a few comments that might be in sections you haven't gotten to yet (apologies if so). Re-request review whenever more files are ready/the whole thing is ready and I'll look again. 

I will say the sandbox isn't working for me (pulls up the docs site, but it doesn't include the OT-2 manual). Not sure if this is a known issue.

jwwojak committed:
this shouldn't be here yet

jwwojak committed:
This section should be removed, for now

jwwojak committed:
Merge branch 'edge' into docs-ot2-manual-revisions

jwwojak committed:
Fix nav and remove link from installation file

Need to remove a file from the nav section of the .yml file and remove link from text in an obsolete file.

jwwojak committed:
Cover HERO render

Found renders. Links in the JIRA ticket.

jwwojak committed:
Minor text change in regulatory.md

jwwojak committed:
docs: OT2 backmatter (#19681)

# Overview

This PR updates and revises the backmatter sections for the OT-2 manual.
It should clearly cover topics similar to the Flex manual.

Sandbox:
https://sandbox.docs.opentrons.com/docs-ot2-revisions-backmatter/

RTC-864

## Changelog

3 new files replace OT-2 ch. 11, Appendix. You only have to review
these:

- `support.md`
- `open-source.md`
- `additional-docs.md`

This also significantly changes the current OT-T end chapter. That
version is an assorted grab-bag of info about Github, our warranty,
unrelated stuff about liquid handling, and just alot of "no, I can't
even..." sort of stuff.

While not a verbatim duplicate of the Flex manual, it is very similar.
Removed some Flex-specific things that aren't relevant to the OT-2.
Still, very boilerplate-ish.

## Review requests

Anything from the current OT-2 manual that's missing and should be in
here? We're going in with a heavy hand.

## Risk assessment

Low, docs; moderate because of the large, installed OT-2 base.

---------

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
docs: ot2 installation and relocation (#19696)

# Overview

This migrates the OT-2 setup instructions into MkDocs. 

Accidentally includes box contents section (`box-contents.md`). Didn't
mean to, but here we are.

Sandbox should be:
https://sandbox.docs.opentrons.com/docs-ot2-install-relocate/

## Test Plan and Hands on Testing

Please review for clarity and accuracy. 

## Changelog

This adds 3 new files (sorry, things get going and I forget to keep
files separate)

- requirements.md: the things you need to have/do before setting up OT-2
- unboxing.md: instructions and images in an ordered list
- box-contents.md: to handle some other text that's been moved out of
the installation section.

Original images are untouched. Many have large padded margins, which
puts almost too much whitespace around them. However, this is good
enough for an old manual. Not sure it's worth it to remove this padding.
They are clear and visible.

## Review requests

Text, grammar, code?

## Risk assessment

Medium.

---------

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Docs OT-2 system description revisions (#20125)

# Overview

This adds a System Description section to the revised OT-2 manual. This
section should contain 3 files:

- components
- pipettes
- robot specs

Sandbox:
https://sandbox.docs.opentrons.com/docs-ot2-system-description-revisions/ot-2/system-description/

RTC-859

## Test Plan and Hands on Testing

Have the usual suspects evaluate the content.

## Changelog

When ready, should be 3 files in a System Description section.

## Review requests

Does the text make sense?
Does the organization make sense?
Are the images clear and relevant?

## Risk assessment

Low - moderate due to complexity of changes.

---------

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
docs: ot2-cleaning-and-maintenance-revisions (#20267)

# Overview

Revises and adds content to a "cleaning and maintenance" section for the
OT-2 manual project.

The cleaning and service sections will closely follow the Flex manual. 

Adding a new 'Self-Service' section to the manual to document some
user-level board maintenance procedures. Note that this section is
_optional_ and can be removed. For the scope of this ticket/task, adding
further end-user repair procedures is out of scope; any additional
self-repair tasks should be addressed in new, separate child tickets.

Pictures and illustrations:
- Images are a mixed bag. Generally potato quality and copies of the
original OT-2 manual. Limited access to a robot means we have less to
work with than with a Flex.
- Using **red to highlight** because Opentrons blue doesn't contrast
enough with dark PCBAs.
- Requesting some new illustrations from Jackson. Marked by placeholder
content.

Sandbox: [Maintenance
section](https://sandbox.docs.opentrons.com/docs-ot2-revisions-cleaning-and-maintenance/ot-2/maintenance/)

RTC-863

## Test Plan and Hands on Testing

Try not to break anything.

## Changelog

New files in a Maintenance chapter:

- `cleaning.md`  cleaning instructions based on the Flex manual.
- `index.md` Intro/landing page
- `self-service.md` Some user-level procedures.
- `service.md` Services offered by Opentrons, based on Flex manual.

## Review requests

- Make sure the text and instructions are clear, links work, and images
make sense.
- Should we save user-level, self-service for a 2nd version?

## Risk assessment

Low, docs changes only.

---------

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
docs: OT2-labware-revisions (#20499)

# Overview

This PR is a revision/replacement for the existing OT-2 Labware chapter.
It closely follows the [Labware
section](https://docs.opentrons.com/flex/labware/) in the Flex manual,
so you can use that for comparison.

**Sandbox:**
https://sandbox.docs.opentrons.com/docs-ot2-revisions-labware/

RTC-861

## Test Plan and Hands on Testing

Reviewed for grammar and link testing. Seems ok, but extra eyes always
help.

## Changelog

Some markdown tables changed to HTML to make them easier to read as
source code. Adds more lines, but clearer (to me at least).

New section: Labware

New files:

- `index.md`: landing page and chapter summary
- `concepts.md`: conceptual discussion of labware
- `types.md`: types of labware the robot can use
- `definitions.md`: JSON stuff

Some images re-cropped to try and improve quality.

## Review requests

Please take a look. Mostly 1:1 replacement from Flex.

## Risk assessment

Low. Docs changes only.

~~In a local build, Mkdocs does show this warning:~~ fixed

```
mkdocs_autorefs: Multiple primary URLs found for 'tips-and-tip-racks':
           ['flex/labware/types/#tips-and-tip-racks', 'ot-2/labware/types/#tips-and-tip-racks']. Make sure to
           use unique headings, identifiers, or Markdown anchors (see our docs).
```

I thought there was some add-on in the .yml file that can resolve this.
The build works though.

---------

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
docs: ot2-pipettes-and-modules-revisions (#20602)

# Overview

This PR does 2 things:

- Adds a top-level TOC section for OT-2 pipettes (1 page). Covers GEN1
and GEN2 pipettes.

- Adds a top level TOC section for OT-2 modules (multiple pages).
Includes heater-shaker, hepa (not hepa/uv), magnetic module (not
magnetic block), temp mod, thermocycler.

Sandbox: 
- Pipettes:
https://sandbox.docs.opentrons.com/docs-instruments-modules-ot2/ot-2/pipettes-ot2/
- Modules:
https://sandbox.docs.opentrons.com/docs-instruments-modules-ot2/ot-2/modules/

RTC-860

## Test Plan and Hands on Testing

Make sure things work properly.

In some places the pages use full path URLs. There will be follow-on
tasks in a later epic to address refactoring links.

## Changelog

New sections for the revised OT-2 manual. The Modules section of the
Flex manual is the model. It's also different from the Flex manual
because it makes the pipette chapter a standalone TOC entry. The Flex
manual has pipettes in the System Description chapter.

## Review requests

- Please review for clarity.
- Any broken links?

## Risk assessment

Low, docs only.

jwwojak committed:
Merge branch 'edge' into docs-ot2-manual-revisions

jwwojak committed:
Restore the docs/mkdocs.yml file

When missing, prevents the sandbox from finding docs and from rendering locally.

jwwojak committed:
Forcing update and autoref check

jwwojak committed:
remove intro

Intro in separate PR

jwwojak committed:
Fixing autorefs to get a build

These can be changed upon merge to main branch.

jwwojak committed:
testing

ecormany reviewed (changes requested):
Did a full pass through the manual. I've listed changes to make page-by-page in this gist: [OT-2 Manual feedback](https://gist.github.com/ecormany/094ced0729b3d9136b337c40ca1b84c9)

jwwojak committed:
docs: ot2 software chapter (#20684)

# Overview

This PR provides an "instruction manual lite" version for the Opentrons
App in the OT-2 manual. It can be seen as a precursor to a longer app
instruction manual for both robots, TBD.

**Edit:** Fixed sandbox link:
https://sandbox.docs.opentrons.com/docs-ot2-manual-software/ot-2/opentrons-app/

RTC-862

## Test Plan and Hands on Testing

Make sure text is accurate, links work, etc.

## Changelog

New software section with pages that will cover the 3 tabs in the
Opentrons App:

- protocols
- labware
- devices

This is a MVP version of what might become a longer, standalone manual.

## Review requests

Please check it out and comment.

## Risk assessment

Low. Docs only.

---------

Co-authored-by: emilyburghardt <emily.burghardt@opentrons.com>

jwwojak committed:
docs: ot2 manual revision introduction (#20756)

# Overview

This creates a new introduction for the revised OT-2 manual. It is based
on the [Flex preface](https://docs.opentrons.com/flex/preface/) from the
manual for that robot. There will be no separate introduction and
preface sections. A simple intro and summary of content is sufficient
for the OT-2.

Edit: fixing autoref errors.

**Edit**: fixed sandbox link:
https://sandbox.docs.opentrons.com/docs-ot2-manual-introduction/ot-2/introduction/

RTC-682

## Test Plan and Hands on Testing

Not much to test. A few summary sections and links.

## Changelog

Adds `introduction.md` to the OT-2 revisions.

## Review requests

- Do these summaries seem alright?

## Risk assessment

- Low, docs only.

---------

Co-authored-by: emilyburghardt <emily.burghardt@opentrons.com>

jwwojak committed:
Chapter organization and file names

Move "installation relocation" before "safety and regulatory."

Fix file names to match page titles.

jwwojak committed:
Page content changes

- Index
- Introduction
- Regulatory
- System description intro
- System description > robot components

jwwojak committed:
Mid-afternoon commit, lots of small changes here

all improvements

jwwojak committed:
End of day

jwwojak committed:
Mid day commit

Based on more reader reviews.

jwwojak committed:
Finishing reviewer comments

jwwojak committed:
Final final first round comments

jwwojak committed:
First run changes

Images. Crop to remove whitespace padding

jwwojak committed:
Mid afternoon

Fixing links and images.

jwwojak committed:
Image and comment change

jwwojak committed:
Fixing a link

Need to change this back to a relative path. Not sure what I did here.

jwwojak committed:
Making reviewer changes from gist

Will update gist with changes.

jwwojak commented on docs/ot-2/docs/images/o-ring-tools.jpeg:
Will remove .jpeg image of o-ring too.

jwwojak committed:
Remove o-ring tool .jpg use .png line art

jwwojak committed:
docs: ot2 manual robot and labware position checks (#20787)

# Overview

Adds a section to cover robot calibration and labware position checks.
Please also see the "Changelog" section.

Sandbox:
https://sandbox.docs.opentrons.com/docs-ot2-calibration-and-lpc/ot-2/calibration/

RTC-938

## Test Plan and Hands on Testing

--

## Changelog

Adds a "Calibration" chapter with these sections:

- `index.md`: intro and landing page
- `robot-calibration.md`: covers robot calibration
- `labware-calibration.md`: covers labware position check
- `jog-controls.md`: covers jog controls

❗ ❗❗❗❗
**Important, please read**: The robot and LPC procedures are
written/presented as procedural steps. However, these calibration
processes are not documented step by step. They're written this way to
show some images and to give readers a sense of what they can expect
when calibrating the robot or using LPC.
❗❗❗❗❗

## Review requests

- Does this cover the topics adequately?

## Risk assessment

Low, docs only.

---------

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Remove placeholders, minor text changes

jwwojak committed:
Small change to gen1 pipette text

Discourage people from trying to use them. Put positive stress on GEN2 models.

jwwojak committed:
Pipette image sizes

This may blur them, but I can't get the sizes reduced with page code. Making originals smaller.

ecormany commented on docs/ot-2/docs/modules/index.md:
```diff
@@ -7,16 +7,16 @@ Modules are a class of deck or externally mounted hardware. You can use an OT-2
 !!! tip
-    - For complete instructions on module installation and use, refer to the guide that shipped with your unit.
-    - For information about integrating modules into your protocols, see the [Protocol Designer Instruction Manual](https://docs.opentrons.com/protocol-designer/) or the [Hardware Modules section](https://docs.opentrons.com/v2/new_modules.html) of our Python API documentation.
+    - For complete instructions on module installation, refer to the guide that shipped with your unit or the [online modules documentation](https://docs.opentrons.com/modules).
```
This can be a relative link.

jwwojak replied:
Fixing that now and updating the shared `.yml` file so OT-2 appears after Flex in the page header.

jwwojak committed:
Add relative link, re-org shared yaml index

This adds a relative link in modules > index to the modules page.

In the shared .yml file, move the OT-2 entry up to appear after Flex.

ecormany reviewed (approved):
Did a full click through and it looks good! Let's get it merged and then figure out the best way to deploy.

jwwojak committed:
Shared card shows Flex and OT-2 on landing page

One card to show 2 robots.
